Sources/Utility/StringExtensions.swift

```diff
@@ -70,4 +70,11 @@ extension String {
 
         return String(cc)
     }
+    
+    /// Splits string with any character in passed-in string of delimiters
+    public func splitWithCharactersInString(delimiters: String) -> [String] {
+        let stops = Set(delimiters.characters)
+        return self.characters.split { stops.contains($0) }.map(String.init)
```

Comment:
Unfortunately that's trying to match the wrong `split` and doesn't compile. I'd have to do
```
characters.split(isSeparator: stops.contains).map(String.init)
```
which reads worse to me.